web/src/views/public/tracks/Show.vue

```diff
@@ -213,11 +263,51 @@ export default {
   .track-page-content__card {
     padding: 24px;
     margin-bottom: 24px;
-    height: 200px;
+    min-height: 200px;
+
+    &--album {
+      padding: 0px;
+      margin-bottom: 12px;
+
+      &--title {
+        border-bottom: rgba(0, 0, 0, 0.05) solid 1px;
+        font-size: 18px;
```

Comment:
This font size is too large. I think `16px` should be good. You can just add the `subtitle-1` class to the `v-card-title` element in the html.